spec/components/provider_interface/sort_order_component_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe ProviderInterface::SortOrderComponent do
+  describe 'rendering' do
+    let(:provider_user) { build_stubbed(:provider_user) }
+    let(:params) do
+      ActionController::Parameters.new({
+        'status' => %w[awaiting_provider_decision pending_conditions],
+        'candidate_name' => 'Susan',
+      })
+    end
+    let(:page_state) do
+      ProviderInterface::ProviderApplicationsPageState.new(params: params, provider_user: provider_user)
+    end
+    let(:rendered_component) { render_inline(component) }
```

Comment:
This shadows `view_component`'s magic `rendered_component` method (like `response` in a request spec). https://github.com/github/view_component#testing-1. Could use a `before` maybe, and leave the tests as is?